its/SonarCloudIntegrationTests.cs

```diff
@@ -0,0 +1,192 @@
+
+            Build currentBuildResult = await ExecuteBuildAndWaitForCompleted(Environment.GetEnvironmentVariable("AZDO_ITS_GRADLE_PIPELINE_NAME"));
+
+            Debug.WriteLine("[Gradle]Build completed.");
+
+            Assert.AreEqual(BuildResult.Succeeded, currentBuildResult.Result);
+            await AssertVersionsOfTasksAreCorrect(currentBuildResult, false);
+
+            await AssertNcLocAndCoverage(Environment.GetEnvironmentVariable("SC_GRADLE_PROJECT_KEY"), 9, 50);
+        }
+
+        [TestMethod]
+        public async Task Execute_Maven_Build_And_Analysis()
+        {
+            Debug.WriteLine("[Maven]Deleting SonarCloud project if it already exists...");
+            //We first delete the project on SonarCloud
+            var isProjectDeleted = await _scInstance.DeleteProjectAsync(Environment.GetEnvironmentVariable("SC_MAVEN_PROJECT_KEY"));
+
+            Assert.AreEqual(true, isProjectDeleted);
+
+            Debug.WriteLine("[Maven]SonarCloud project has been successfully deleted...");
+            Debug.WriteLine("[Maven]Queuing the corresponding build and waiting for its completion...");
+
+            Build currentBuildResult = await ExecuteBuildAndWaitForCompleted(Environment.GetEnvironmentVariable("AZDO_ITS_MAVEN_PIPELINE_NAME"));
+
+            Debug.WriteLine("[Maven]Build completed.");
+
+            Assert.AreEqual(BuildResult.Succeeded, currentBuildResult.Result);
+            await AssertVersionsOfTasksAreCorrect(currentBuildResult, false);
+
+            await AssertNcLocAndCoverage(Environment.GetEnvironmentVariable("SC_MAVEN_PROJECT_KEY"), 211, 23.1);
+        }
+
+        [TestMethod]
+        public async Task Execute_Dotnet_Build_And_Analysis()
+        {
+            Debug.WriteLine("[Dotnet]Deleting SonarCloud project if it already exists...");
+            //We first delete the project on SonarCloud
+            var isProjectDeleted = await _scInstance.DeleteProjectAsync(Environment.GetEnvironmentVariable("SC_DOTNET_PROJECT_KEY"));
+
+            Assert.AreEqual(true, isProjectDeleted);
+
+            Debug.WriteLine("[Dotnet]SonarCloud project has been successfully deleted...");
+            Debug.WriteLine("[Dotnet]Queuing the corresponding build and waiting for its completion...");
+
+            Build currentBuildResult = await ExecuteBuildAndWaitForCompleted(Environment.GetEnvironmentVariable("AZDO_ITS_DOTNET_PIPELINE_NAME"));
+
+            Debug.WriteLine("[Dotnet]Build completed.");
+
+            Assert.AreEqual(BuildResult.Succeeded, currentBuildResult.Result);
+
+            await AssertVersionsOfTasksAreCorrect(currentBuildResult);
+
+            await AssertNcLocAndCoverage(Environment.GetEnvironmentVariable("SC_DOTNET_PROJECT_KEY"), 43, 25);
+        }
+
+        private static VssConnection GetAzDoConnection()
+        {
+            VssBasicCredential credentials = new VssBasicCredential(string.Empty, Environment.GetEnvironmentVariable("AZURE_TOKEN"));
```

Comment:
I think you can move the `Environment.GetEnvironmentVariable` calls into static final variables to improve readability

Reply:
@andrei-epure-sonarsource @duncanp-sonar i did some change to centralize and have some dedicated classes to manage those values; should be easier to read now.